MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/Utilities/MarsDate.cs

```diff
@@ -65,10 +65,35 @@ public static double MarsToEarthDate(DateTime marsDate)
             return earthSolDate;
 
         }
-        public static double RoverSol(DateTime roverDate)
+        //the time the different rovers have been on mars 
+        public static double CuriositySol(DateTime roverDate)
         {
-            double differenceInDays = (double)roverDate.Subtract(new DateTime(2012, 08, 06)).TotalDays;
-            double roverDays = (double)(differenceInDays * earthConversionRate);
+            double differenceInDays = (int)roverDate.Subtract(new DateTime(2012, 08, 06)).TotalDays;
+            double roverDays = (int)(differenceInDays * earthConversionRate);
+            return roverDays;
+        }
+        public static double PerseveranceSol(DateTime roverDate)
+        {
+            double differenceInDays = (int)roverDate.Subtract(new DateTime(2021, 02, 18)).TotalDays;
+            double roverDays = (int)(differenceInDays * earthConversionRate);
```

Comment:
note that when you cast a double to an integer, I think all that happens is that C# will throw away all the values after the decimal point.

ie if you do `(int)4.8` it will give you back `4` rather than rounding it to 5.
Not sure if that's what you want or not?

If you want to round first, then you'll need to use the `Math.Round` method.
So something like `(int)Math.Round(yourNumber)`